docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -848,7 +848,9 @@ North Carolina, filename VR_Snapshot_20220101.txt; see
 2022_06_02b_prl_code_for_probs_of_spaces_and_hyphens_in_last_and_first_names.ipynb
 for computation details.)
 
-
+For now, assign all simulants within the same household the same last name. 
+This is an oversimplification as some might not be related, but will work 
+for the simple model. 
```

Comment:
it would not be too complicated for us to exclude anyone with "relation to head of household" "roommate" or "other nonrelative".

i also assume (and if so can you include) that you don't want this to be done to anyone in group quarters?

Reply:
Yeah, I will add GQ for sure! Good catch. We were discussing roommate/other non-relative. If it is relatively easy I can add it in